src/octokit-helper.ts

```diff
@@ -0,0 +1,23 @@
+import * as github from '@actions/github'
+import {Octokit} from '@octokit/rest'
+import {getServerApiUrl} from './url-helper'
+
+// Centralize all Octokit references by re-exporting
+export {Octokit} from '@octokit/rest'
+
+export type OctokitOptions = {
+  baseUrl?: string
+  userAgent?: string
+}
+
+export function getOctokit(authToken: string, opts: OctokitOptions) {
```

Comment:
Sure, I just went with `-helper` which was consistent with the other files. I did also move the `@octokit/rest` import and export into this file too so that all octokit references are made from this file keeping it all together.

I have renamed it to `-provider`